### ISSUE TITLE
[Feature request] Allow searching lora by version name

### ISSUE THREAD

**Issue by DJLegends1011** [Kinglord/ComfyUI_LoRA_Sidebar#10]
noticed that you can search via tag but it would be nice to search via version name since sometime authors have 10+ loras stuck under one page and this node can't differentiate between which is which so the user need to add custom tags

**Comment by DJLegends1011** [comment]
learning it the hard way after finding a page hosting 90 Loras and it has the same name :(

**Comment by Kinglord** [comment]
Yeah this is a pattern that I've seen getting worse and not better, it breaks how the CivitAI api endpoints like to work which makes dealing with it a PITA but I will see what I can do.

**Comment by DJLegends1011** [comment]
> Yeah this is a pattern that I've seen getting worse and not better, it breaks how the CivitAI api endpoints like to work which makes dealing with it a PITA but I will see what I can do.

oh I didn't knew that
if anything i'd take you fixing tags not being persisted on restarts

**Comment by Kinglord** [comment]
> > Yeah this is a pattern that I've seen getting worse and not better, it breaks how the CivitAI api endpoints like to work which makes dealing with it a PITA but I will see what I can do.
> 
> oh I didn't knew that if anything i'd take you fixing tags not being persisted on restarts

What do you mean not persisted? If you add or remove tags from a lora that should be updating the data and be persistent. What are you doing that isn't saved between restarts?
